kano_greeter/user_list.py

```diff
@@ -99,8 +99,8 @@ def _btn_add_account_pressed(self, event=None, button=None):
         win.go_to_newuser()
 
     def _btn_shutdown_pressed(self, event=None, button=None):
-        shutdown_dialog = KanoDialog(title_text='Shutting down..',
-                                     description_text='Are you sure you want to shutdown your Kano now?',
+        shutdown_dialog = KanoDialog(title_text=_('Shutting down..'),
```

Comment:
KanoDialog is in a different module, isn't it? Do we need  a debian dependency on the version which supports this?